stream-video-android-core/src/androidTest/kotlin/io/getstream/video/android/core/LivestreamTest.kt

```diff
@@ -89,60 +85,26 @@ class LivestreamTest : IntegrationTestBase() {
             println("hi123 event: $it")
         }
         call.join(create = true)
+        print("debugging: call cid is:  " + call.cid)
         Thread.sleep(1000L)
 
-        // counts
-        val session = call.state.session.value
-        assertThat(session?.participants).isNotEmpty()
-        assertThat(session?.startedAt).isNotNull()
-
-        assertThat(session?.participantsCountByRole).isNotEmpty()
+        // counts and startedAt
+        assertThat(call.state.participants.value).isNotEmpty()
+        assertThat(call.state.startedAt).isNotNull()
+        assertThat(call.state.participantCounts.value?.total).isEqualTo(1)
+        assertThat(call.state.participantCounts.value?.anonymous).isEqualTo(0)
     }
 
     @Test
-    @Ignore
     fun timeRunning() = runTest {
```

Comment:
Running just this test alone (from AS) will result in:
```
kotlinx.coroutines.TimeoutCancellationException: Timed out waiting for 10000 ms
at kotlinx.coroutines.TimeoutKt.TimeoutCancellationException(Timeout.kt:191)
```
but it runs fine if you run all the LiveStreamTest tests at  once. I think it relies on something / some state from previous tests.

Reply:
But sometimes it runs fine 🤷‍♂️ . Something is flaky here.

Reply:
Ok, this is solved with my PR - so after locally rebasing with `develop` it runs fine.